examples/cluster/small.yaml

```diff
@@ -29,7 +29,7 @@ Strategy:
             PostInstallHook:
               - Execute:
                   Path: "."
-                  Shell: "gcloud container clusters get-credentials testcluster --region=us-east4 --project=beamery-trials"
+                  Shell: "pwd"
```

Comment:
Removing this means the kubectx will no longer be set for application deployment?

Reply:
This part will be manage by my code.
Every time, gravitywell will check if it found a context with the format "{ProjectName}_{ClusterName}_{Region}". If it did not found it, it will going to set the context with the cluster information. Currently, it only working for google cloud provider